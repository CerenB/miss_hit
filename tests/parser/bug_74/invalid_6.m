% (c) Copyright 2020 Florian Schanda

[x;y]' = size(1);
