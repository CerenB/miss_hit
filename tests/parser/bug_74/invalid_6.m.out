In invalid_6.m, line 3
| [x;y]' = size(1);
|        ^ error: left-hand side of assignment must be a Name, found Unary_Operation instead
MISS_HIT Summary: 1 file(s) analysed, 1 error(s)
